tests/page-objects/messages/sms-gateway.po.js

```diff
@@ -0,0 +1,125 @@
+const utils = require('../../utils');
+const commonElements = require('../common/common.po');
+const helper = require('../../helper');
+const { browser, element } = require('protractor');
+
+//page objects        
+const  sentTask= element(by.css('#reports-content .details > ul .task-list .task-state .state'));
+   
+const deliveredTask = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(1) > ul > li:nth-child(1) .task-state .state'));
+
+const scheduledTask = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(1) > ul > li:nth-child(2) .task-state .state'));
+      
+const failedTask = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(2) > ul > li:nth-child(1) .task-state .state'));
+    
+const feedback = element(
+  by.css('#reports-content .details > ul .task-list .task-state .state')
+);
+
+ 
+// scheduled tasks
+// State for messageId2 is still forwarded-to-gateway
+const message2State = element(by.css(
+  '#reports-content .scheduled-tasks > ul > li:nth-child(1) > ul > li:nth-child(1) .task-state .state'));
+
+const getTaskState = element => {
+  helper.waitUntilReady(element);
+  return helper.getTextFromElement(element);
+}; 
+
+module.exports = {
+  
+  showMessageList : async() => {
+    utils.resetBrowser();
+    helper.clickElement(element(by.id('messages-tab')));
+  
+    // LHS
+    helper.waitElementToPresent(
+      element(by.css('#message-list li:first-child'))
+    );
+    browser.waitForAngular();
+    return await helper.waitElementToBeVisible(
+      element(by.css('#message-list li:first-child'))
+    );
+  }, 
+
+  expectMessage: async(heading, summary) => {
+    await expect(
+      helper.getTextFromElement(
+        element(by.css('#message-list li:first-child .heading h4'))
+      )
+    ).toBe(heading);
+    await expect(
+      helper.getTextFromElement(
+        element(by.css('#message-list li:first-child .summary p'))
+      )
+    ).toBe(summary);
+  },
+
+  showMessageDetails: async() => {
+    // RHS
+    helper.clickElement(
+      element(by.css('#message-list li:first-child .summary'))
+    );
+    helper.waitElementToBeVisible(
+      element(
+        by.css('#message-content li.incoming:first-child .data p:first-child')
```

Comment:
There are a few css identifier is used multiple times. It could be a bit cleaner by either setting the element as a variable above or in the module.exports depending on use. 

`const incomingData = element(by.css('#message-content li.incoming:first-child .data p:first-child'))`